#----------------------
# Serwo info
#----------------------
float32[] matrix
int32 found
int32 out_time_nsec_pocz
int32 out_time_sec_pocz
int32 out_time_nsec_kon
int32 out_time_sec_kon
